src/languages/en-US.ts

```diff
@@ -2250,6 +2257,37 @@ export default class extends Language {
 			extendedHelp: `This command shows a random starred message or the starboard usage and leaderboard for this server.`
 		}),
 
+		/**
+		 * ##################
+		 * SUGGESTIONS COMMANDS
+		 */
+
+		COMMAND_SUGGEST_DESCRIPTION: 'Hey admins! Please do this and that...',
+		COMMAND_SUGGEST_EXTENDED: builder.display('suggest', {
+			extendedHelp: 'Posts a suggestion to the server\'s suggestion channel',
+			explainedUsage: [
+				['suggestion', 'Your suggestion']
+			],
+			examples: [
+				'let\'s make a music channel'
+			],
+			reminder: 'You need to have setup a suggestions channel for this command to work. If you are an administrator, you will be given the chance to do so upon invoking the command'
+		}),
+		COMMAND_RESOLVESUGGESTION_DESCRIPTION: 'Mark a suggestion as accepted, considered, or even denied',
+		COMMAND_RESOLVESUGGESTION_EXTENDED: builder.display('resolvesuggestion', {
```

Comment:
```suggestion
		COMMAND_RESOLVESUGGESTION_EXTENDED: builder.display('resolveSuggestion', {
```

Reply:
this isn't needed, and it looks... off, compared to other 2 word commands (for example, escaperope)

Reply:
escape rope should be displayed as escapeRope, most multi-word commands use camelCase convention for readability.